crates/next-core/js/src/overlay/internal/helpers/clsx.ts

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
In future we need to figure out some kind of build process that doesn't need to inline these package into your repo.

Maybe bundling (it would be cool to use turbopack for that, but we don't have production builds yet)

Reply:
Yes, I think we'll need to bundle if we move it into a package as well